### ISSUE TITLE
The results included the distances between sequence and itself

### ISSUE THREAD

**Issue by Huanchang-Yan** [veg/hivtrace#80]
Dear @stevenweaver ,

Hello.

I ran the following command line without any error reported:

hivtrace -i XX.fasta -a resolve -r HXB2_pol -t .015 -m 500 -g .05

However, the results in the csv file included the distance between every sequence and itself.

When viewing the JSON file, it failed.
![捕获](https://user-images.githubusercontent.com/65205967/133041808-6a058dfc-6425-47d8-b1ee-3c1af15b54c1.JPG)

The fasta file includes 66 sequences without any duplicates.

I have no clue so I am writing to ask for your favor.

Many thanks.

Best wishes,
Huanchang

**Comment by spond** [comment]
Dear @Huanchang-Yan,

This behavior simply means that you have no clusters in your dataset. The visualization module is showing you an empty network. Switch over to the `Statistics` tab and confirm.

Best,
Sergei

**Comment by Huanchang-Yan** [comment]
Dear @spond ,

Thanks for your response.

Actually, there are some pairwise distances below the threshold of 0.015. 
I think, the failure of the network visualization might be due to the results that contains pairwise distance between every sequence and itself, just like this in output.csv:
![1 (2)](https://user-images.githubusercontent.com/65205967/133089898-65806d20-49b9-4b82-b627-576deb705e5e.JPG)
Also, I tried other methods such as Hyphy and MicrobeTrace, and used the same threshold, and I did find some clusters. But I have no clue to get it correctly in HIVTRACE.

Best wishes,
Huanchang
